packages/sheriff-create-config/eslint.config.ts

```diff
@@ -11,8 +11,12 @@ const sheriffOptions: SheriffSettings = {
   vitest: false,
 };
 
-export default tseslint.config(sheriff(sheriffOptions), {
-  rules: {
-    'fsecond/prefer-destructured-optionals': 0,
+export default tseslint.config(
+  sheriff(sheriffOptions),
+  {
+    rules: {
+      'fsecond/prefer-destructured-optionals': 0,
+    },
   },
-});
+  { ignores: ['tsdown.config.ts'] },
```

Comment:
i think this is not needed. We probably just need to include `tsdown.config.ts` in the `tsconfig.json`